apps/game_backend/lib/game_backend/users/user.ex

```diff
@@ -64,10 +64,10 @@ defmodule GameBackend.Users.User do
       :profile_picture,
       :google_user_id
     ])
-    |> unique_constraint([:game_id, :username])
     |> cast_assoc(:unlocks)
     |> assoc_constraint(:google_user)
     |> validate_required([:game_id, :username])
+    |> validate_length(:username, min: 3, max: 9)
```

Comment:
Minimum should be 2.
```suggestion
    |> validate_length(:username, min: 2, max: 9)
```